build/commands/lib/util.js

```diff
@@ -420,6 +457,71 @@ const util = {
     }
   },
 
+  touchOverriddenFiles: (excludeTests = true) => {
+    console.log('touch original files overridden by chromium_src...')
+
+    // Return true when original file of |file| should be touched.
+    const applyFileFilter = (file) => {
+      // Exclude test files
+      if (excludeTests && (file.indexOf('browsertest') > -1 || file.indexOf('unittest') > -1)) { return false }
```

Comment:
that's an amazing PR! It also bugged me that we don't do this for tests :)

just a quick question: do we really need `excludeTests` var? does it speed up things dramatically? I think it's okay to touch everything, should be unnoticeable.

Reply:
The optimization was already there so I kept it, but I think you are right: not likely to be a significant time saver.

Reply:
Updated in [7152be9](https://github.com/brave/brave-core/pull/13597/commits/7152be982f72b1ea7fff5ae5b027eb77c850f9fa)